test/test_entanglement_tracker.jl

```diff
@@ -114,3 +114,136 @@ for i in 1:30, n in 2:30
     @test observable((q1.slot, q2.slot), Z⊗Z) ≈ 1
     @test observable((q1.slot, q2.slot), X⊗X) ≈ 1
 end
+
```

Comment:
let's make all of this a separate file -- I want to be able to easily run a single file as a tests (when working interactively)